### DISCUSSION TITLE
Welcome to custom-angular-calendar Discussions!

### DISCUSSION THREAD

**Discussion by kartiknagpal2828** [kartiknagpal2828/custom-angular-calendar#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JBP1996** [comment]
hello I have downloaded your calendar, and I already have checked a minor bug. I feel that was a good project.
I hope I can help with some ideas and some developments in this repo if you want of course 👍 .
I think one good feature to add is set some default holidays that are common in all the world, and next for each country and so on.

I will follow this project, nice job 💪

**Reply by kartiknagpal2828** [reply]
Sure. That would be great. 
Let me know if you have any questions about this.

**Reply by JBP1996** [reply]
@kartiknagpal2828  I have a pull request now. I don't know how do you want to organize the project. but if you give me permissions to send changes to your repo porbably was more easy to I work in a fork from it and make allways new pull requests from a fork from outside than a branch from inside. Any question or talk send me an email for: jbp.1996@hotmail.com

Regards ;)
